### DISCUSSION TITLE
Property and Method links in comments

### DISCUSSION THREAD

**Discussion by InkONat** [godotengine/godot-proposals#8701] [Engine Core]
You should be able to put links to properties and methods in the current script (or to some other script or custom class).

The way you make the link should be discussed, but my idea is 

```
# (Text that will be displayed as a link)[<class or script file path, self if its for this script> <property or method name>]
```

For example, if I want to add a link to the method `my_method` of the class `MyClass`, i can do `# (Method my_method of class MyClass)[MyClass my_method]`, which will display on the script editor `# Method my_method of class MyClass`, and there will be a line below that text when your cursor hovers over it.

Maybe you could also link to doc pages as well?

- *Production edit: Continued from https://github.com/godotengine/godot-proposals/discussions/8697.*

**Comment by aXu-AP** [comment]
Note that we already have a syntax for this (it's used by generated documentation): https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_documentation_comments.html
Would just need to add editor support for making those links clickable from script editor in addition to documentation pages.
I think that if you refer to a member from a script, it should take you to the script, but for built-in members open the docs instead.
